packages/core/src/components/account/MyAccountMenu/MyAccountMenu.tsx

```diff
@@ -0,0 +1,85 @@
+import { Button, Avatar, Link } from '@faststore/ui'
+import { useRouter } from 'next/router'
+
+import styles from '../section.module.scss'
```

Comment:
I think we can create a css file for each component and import this file inside `section.module.scss` . Did u try it?